docs/projections/projections-config.md

```diff
@@ -0,0 +1,109 @@
+
+![Web admin interface projections configuration screen](../images/wai-projection-config.jpg)
+
+<!-- TODO: Further explanation here -->
+
+<<< @/docs/server/v5/http-api/sample-code/projections/create-projection.sh
+
+## Emit options
+
+These options control how projections append events.
+
+In busy systems, projections can put a lot of extra pressure on the master node. This is especially true for EventStoreDB servers that also have persistent subscriptions running, which only the master node can process. If you see a lot of commit timeouts and slow writes from your projections and other clients, then start with these settings.
+
+### Emit enabled
+
+The `emit` boolean setting determines whether a projection can emit events and any projection that calls `emit()` or `linkTo()` requires it. If this option is not set and a projection attempts to emit events, you see an error message like the following:
+
+<!-- TODO: Is it emit or emitenabled? Or are .NET and HTTP different -->
+
+```bash
+'emit' is not allowed by the projection/configuration/mode
+```
+
+EventStoreDB disables this setting by default, and is usually set when you create the projection and if you need the projection to emit events.
+
+### Track emitted streams
+
+The `trackemittedstreams` boolean setting enables tracking of a projection's emitted streams. It only has an affect if the projection has `EmitEnabled` enabled.
+
+Tracking emitted streams enables you to delete a projection and all the streams that it has created. You should only the setting if you intend to delete a projection and create new ones that project to the same stream.
+
+::: warning
+By default, EventStoreDB disables the `trackemittedstreams` setting for projections. When enabled, an event appended records the stream name (in `$projections-{projection_name}-emittedstreams`) of each event emitted by the projection. This means that write amplification is a possibility, as each event that the projection emits appends a separate event. As such, this option is not recommended for projections that emit a lot of events, and you should enable only where necessary.
+:::
+
+::: tip
+Between EventStoreDB versions 3.8.0 and 4.0.2, this option was enabled by default when a projection was created through the UI. If you have any projections created during this time frame, it's worth checking whether this option is enabled.
+:::
+
+### Max allowed writes in flight
+
+<!-- TODO: Why is this not in the GUI for new projection? -->
+
+<!-- TODO: Is the setting name correct? Especially in HTTP -->
+
+The `AllowedInFlightMessages` setting sets the maximum number of writes to allow for a projection. Because a projection can write to multiple different streams, it's possible for the projection to send multiple writes at the same time. This option sets the number of concurrent writes that a projection can perform.
```

Comment:
The Http API accepts `MaxAllowedWritesInFlight`

```suggestion
The `MaxAllowedWritesInFlight` setting sets the maximum number of writes to allow for a projection. Because a projection can write to multiple different streams, it's possible for the projection to send multiple writes at the same time. This option sets the number of concurrent writes that a projection can perform.
```